scripts/tail-crashlog.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+## BEREKEN TOTAAL AANTAL BOEKEN ##
+
+tail -n 600 ./crash.log  > ./scripts/tail-crashlog.txt
```

Comment:
```suggestion
## BEREKEN TOTAAL AANTAL BOEKEN ##

tail -n 600 ./crash.log >./scripts/tail-crashlog.txt
```

(Although does this require a shell invocation?)

Reply:
I vaguely remember mentioning using `os.exec` instead of shell one-liners in the past, yeah ^^.